core/spam/vote_spam_policy.go

```diff
@@ -433,3 +434,20 @@ func (vsp *VoteSpamPolicy) PreBlockAccept(tx abci.Tx) (bool, error) {
 
 	return true, nil
 }
+
+func (vsp *VoteSpamPolicy) GetStats(partyID string) Statistic {
+	vsp.lock.RLock()
+	defer vsp.lock.RUnlock()
+
+	stats := Statistic{}
+
+	bStats, ok := vsp.partyBlockRejects[partyID]
+	if !ok {
+		return stats
+	}
+
+	stats.Total = strconv.FormatUint(bStats.total, formatBase)
```

Comment:
it might also be worth returning the actual min tokens required as this may be growing during a perceived attack